### PR TITLE
Fix for wrongly created stubs configuration

### PR DESCRIPTION
**Pull request by adrian-wozniak**
Workaround for bug in spring-cloud-zookeeper: https://github.com/spring-cloud/spring-cloud-zookeeper/issues/138

### CODE REVIEW

File: micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/discovery/ZookeeperDependencyStubsConfigurationFix.groovy [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                if (!dependency.stubs) {
+                    ServicePath servicePath = new ServicePath(dependency.path)
+                    MicroserviceConfiguration.Dependency.StubsConfiguration stubConfiguration =
+                            new MicroserviceConfiguration.Dependency.StubsConfiguration(servicePath)
+
+                    dependency.stubs = stubConfiguration.toColonSeparatedDependencyNotation()
+                    dependency.stubsConfiguration = new StubsConfiguration(
+                            stubConfiguration.stubsGroupId,
+                            stubConfiguration.stubsArtifactId,
+                            stubConfiguration.stubsArtifactId
```
Comment (yu55) [inline micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/discovery/ZookeeperDependencyStubsConfigurationFix.groovy]: Duplicated stubConfiguration.stubsArtifactId ?

File: micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/discovery/ZookeeperDependencyStubsConfigurationFix.groovy [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import org.springframework.cloud.zookeeper.discovery.dependency.ZookeeperDependency
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Profile
+
+/**
+ * Workaround for bug in spring-cloud-zookeeper
+ * https://github.com/spring-cloud/spring-cloud-zookeeper/issues/138
+ */
+@Configuration
+@Profile(BasicProfiles.SPRING_CLOUD)
```
Comment (yu55) [inline micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/discovery/ZookeeperDependencyStubsConfigurationFix.groovy]: Static import would look more concise.

Comment (adrian-wozniak) [inline micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/discovery/ZookeeperDependencyStubsConfigurationFix.groovy]: As you wish :)

File: micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/discovery/ZookeeperDependencyStubsConfigurationFix.groovy (hunk 1 above)
Comment (adrian-wozniak) [inline micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/discovery/ZookeeperDependencyStubsConfigurationFix.groovy]: That's mistake, but all in all `stubsConfiguration` is overridden during `ZookeeperDependencies` post construct. Removing this line.

**Pull request merged**